src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -150,44 +153,50 @@ private void initializeRamseteCommands() {
   }
 
   private void initializeCommandGroups() {
-    // TODO: the wait time should not be a constant, should be configurable
 
     taxiCommand =
         new SequentialCommandGroup(
-            new WaitCommand(Constants.delaytaxi),
+            new WaitCommand(initialWaitTime),
             // taxiRamseteCommands[0]);
-            taxiRamseteCommand.beforeStarting(
-                () -> driveBaseSubsystem.resetOdometry(taxiTrajectory.getInitialPose())));
+            taxiRamseteCommand
+                .beforeStarting(
+                    () -> driveBaseSubsystem.resetOdometry(taxiTrajectory.getInitialPose()))
+                .andThen(() -> driveBaseSubsystem.stopDriveMotors()));
 
     if (allSubsystemsEnabled) {
       oneBallCommand =
           new SequentialCommandGroup(
-              new WaitCommand(Constants.delayshot),
+              new WaitCommand(initialWaitTime),
               // new ShooterPrime(shooterSubsystem, limelightSubsystem, cdsSubsystem),
               new ShooterPressed(
                   shooterSubsystem,
                   limelightSubsystem,
                   cdsSubsystem,
                   true), // also has a time delay of 2-3 seconds
               new WaitCommand(Constants.delaytaxi),
-              oneBallRamseteCommand.beforeStarting(
-                  () -> driveBaseSubsystem.resetOdometry(oneBallTrajectory.getInitialPose())));
+              oneBallRamseteCommand
+                  .beforeStarting(
+                      () -> driveBaseSubsystem.resetOdometry(oneBallTrajectory.getInitialPose()))
+                  .andThen(() -> driveBaseSubsystem.stopDriveMotors()));
 
       twoBallParallel =
           new ParallelDeadlineGroup(
               twoBallRamseteCommand.beforeStarting(
                   () ->
                       driveBaseSubsystem.resetOdometry(
```

Comment:
This is the only autonomous mode you have so far that doesn't stop drive motors after resetting odometry. Was that intentional?

Reply:
No, I forgot to add that. Thanks for the catch!